src/components/menus/directory-tree-contextual-menu.js

```diff
@@ -360,6 +366,14 @@ const DirectoryTreeContextualMenu = (props) => {
             icon: <FolderSpecialIcon fontSize="small" />,
         });
 
+        menuItems.push({
+            messageDescriptorId: 'StashedElements',
+            callback: () => {
+                handleOpenDialog(DialogsId.STASHED_ELEMENTS);
+            },
+            icon: <RestoreFromTrash fontSize="small" />,
+        });
```

Comment:
Maybe disable the option if the bin is empty ? Like it is done for modifications and nodes

Reply:
done